titles/admin-rhdh/title-admin.adoc

```diff
@@ -16,6 +16,9 @@ include::modules/admin/proc-install-operator.adoc[leveloffset=+1]
 // ocp deployment
 include::assemblies/assembly-install-rhdh-ocp.adoc[leveloffset=+1]
 
+// configure PostgreSQL database
+include::assemblies/assembly-configuring-external-postgresql-databases.adoc[leveloffset=+1]
```

Comment:
Does this process apply only to admins or to other users as well? If it applies to other users, can we include this content in the Getting Started guide until we have a dedicated title for it?

Reply:
Yes, i think both the adminstrator and developer role users can configure an external PostgreSQL instance. @zdrapela, tagging you for confirmation. 

If this is the case, do you mean we need to include this content both in admin and getting started guides, @hmanwani-rh?

Reply:
Perhaps a reference in the Getting Started guide would also be helpful.

Reply:
Both the adminstrator and developer in OpenShift can configure external PostgreSQL, but I believe that this should be only in the Administration/Installation guide. The Getting started already points to the Administration guide for the installation.